timeline/tests/test_templatetags.py

```diff
@@ -180,6 +180,39 @@ def test_get_app_icon_html_plugin(self):
         self.assertIn(plugin.title, ret)
         self.assertIn(url, ret)
 
+    def test_get_app_icon_html_example_site_app(self):
+        """Test get_app_icon_html() on event from an app plugin"""
+        event = self._make_event(
+            project=self.project,
+            app='example_site_app',
+            user=self.user_owner,
+            event_name=TEST_EVENT_NAME,
+            description=TEST_EVENT_DESC,
+        )
+        plugin = get_app_plugin('example_site_app')
+        url = reverse(
+            plugin.entry_point_url_id,
+            kwargs={'project': self.project.sodar_uuid},
+        )
+        ret = tags.get_app_icon_html(event, self.plugin_lookup)
+        self.assertIn(plugin.icon, ret)
+        self.assertIn(plugin.title, ret)
+        self.assertIn(url, ret)
```

Comment:
We should also assert that `url` corresponds to the url reversed from the `entry_point_url_id` of the plugin. Other than that the test case looks very good.